src/flood_forecaster/data_ingestion/load.py

```diff
@@ -0,0 +1,50 @@
+# Methods to load the data from the database
+from datetime import datetime
+
+import pandas as pd
+from sqlalchemy import select
+
+from src.flood_forecaster.data_model.river_level import HistoricalRiverLevel
+from src.flood_forecaster.data_model.weather import HistoricalWeather, ForecastWeather
+from src.flood_forecaster.utils.configuration import Config
+from src.flood_forecaster.utils.database_helper import DatabaseConnection
+
+
+def load_history_weather_db(config: Config, location: str, date_begin: datetime, date_end: datetime):
+    """
+    Loads the history weather data from the database and returns it as a pandas dataframe
+    :param config:
+    :param location:
+    :param date_begin:
+    :param date_end:
+    :return: pandas dataframe
+    """
+    stmt = (select(HistoricalWeather)
+            .where(HistoricalWeather.location_name == location)
+            .where(HistoricalWeather.date > date_begin)
+            .where(HistoricalWeather.date < date_end))
+    database = DatabaseConnection(config)
+    return pd.read_sql(stmt, database.engine)
+
+
+def load_forecast_weather_db(config: Config, location: str, date_begin: datetime, date_end: datetime):
+    stmt = (select(ForecastWeather)
+            .where(HistoricalWeather.location_name == location)
```

Comment:
Shouldn't it be `ForecastWeather` instead of `HistoricalWeather` here ? Same below :)

Reply:
oops, copy-paste error